modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase

```diff
@@ -26,10 +47,15 @@ definition {
 		if (${testPortalInstance} == "true") {
 			PortalInstances.tearDownCP();
 
+			TestUtils.hardRefresh();
+
```

Comment:
Refreshing the page doesn't clear the browser cache.

Reply:
You may want to try closing the browser window and opening incognito window.

Reply:
Right... I thought that hard refresh would also empty cash, thanks for clarifying...

Reply:
[Please see it](https://liferay.slack.com/archives/CD7939WBE/p1648168045484689?thread_ts=1648061188.934189&cid=CD7939WBE)